dashboard/app/models/single_facilitator_experiment.rb

```diff
@@ -0,0 +1,40 @@
+# == Schema Information
+#
+# Table name: experiments
+#
+#  id                   :integer          not null, primary key
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#  name                 :string(255)      not null
+#  type                 :string(255)      not null
+#  start_at             :datetime
+#  end_at               :datetime
+#  section_id           :integer
+#  min_user_id          :integer
+#  max_user_id          :integer
+#  overflow_max_user_id :integer
+#  earliest_section_at  :datetime
+#  latest_section_at    :datetime
+#  script_id            :integer
+#
+# Indexes
+#
+#  index_experiments_on_max_user_id           (max_user_id)
+#  index_experiments_on_min_user_id           (min_user_id)
+#  index_experiments_on_overflow_max_user_id  (overflow_max_user_id)
+#  index_experiments_on_section_id            (section_id)
+#
+
+class SingleFacilitatorExperiment < Experiment
```

Comment:
I don't see anything here that's unique to facilitators. Unless you're planning to add some special facilitator logic later, could you rename this to `SingleUserExperiment`?